python-avd/pyavd/_eos_designs/shared_utils/utils.py

```diff
@@ -72,3 +72,7 @@ def get_merged_adapter_settings(self: SharedUtils, adapter_or_network_port_setti
         profile_name = adapter_or_network_port_settings.get("profile")
         adapter_profile = self.get_merged_port_profile(profile_name)
         return merge(adapter_profile, adapter_or_network_port_settings, list_merge="replace", destructive_merge=False)
+
+    def get_ip_from_ip_prefix(self: SharedUtils, ip_prefix: str) -> str:
+        """Return the ip part of an ip/mask prefix."""
+        return ip_prefix.split("/", maxsplit=1)[0]
```

Comment:
Can we use the python module to get the ip address from ip prefix
```
from ipaddress import ip_interface
return (str(ip_interface(ip_prefix).ip))
```

Reply:
could do - I guess it all depends which one would be faster

Reply:
yes current one is more faster then ip_interface